constants.py

```diff
@@ -64,14 +64,20 @@
 
 
 PROMPT_TEXT_ANT_CHOICE  = "please enter your choice of ant"
-"""The text which will be contained in the question when asking the user to choose which ant to move.
+"""The text which will be contained in the question when asking the user to choose which 
+ant to move.
```

Comment:
why the line split?

Reply:
The lack of line break in previous versions was an oversight. It did not fit on the screen properly as it was and now it does; also now it matches the other PROMPT_TEXTs.